boot/zephyr/Kconfig

```diff
@@ -28,6 +27,24 @@ partition=MCUBOOT_PAD
 partition-size=0x200
 source "${ZEPHYR_BASE}/../nrf/subsys/partition_manager/Kconfig.template.partition_size"
 
+config PM_PARTITION_SIZE_MCUBOOT
+	hex
+	default FPROTECT_BLOCK_SIZE if (SOC_NRF5340_CPUAPP || SOC_NRF9160)
+	default 0x3e00
+	depends on (BOOT_NRF_EXTERNAL_CRYPTO && BOOT_MIN_PARTITION_SIZE)
+
+config PM_PARTITION_SIZE_MCUBOOT
+	hex
+	default FPROTECT_BLOCK_SIZE if SOC_NRF9160
+	default 0x7c00
```

Comment:
Note, this will not be updated if user later sets `BOOT_MIN_PARTITION_SIZE=y` using menuconfig or directly in `.config`.

See more here: https://docs.zephyrproject.org/latest/guides/kconfig/tips.html#stuck-symbols-in-menuconfig-and-guiconfig